source/faucet/config.yaml

```diff
@@ -2,6 +2,15 @@
 ##                                  Sample keys                               ##
 ################################################################################
 
+# How frequently we run our periodic task in seconds
+interval: 1
+
+# Between how many addresses we split a transaction by
+count: 1
+
+# Maximum number of utxo before merging instead of splitting
+max_utxos: 10
+
```

Comment:
Two things:
1) Make it part of a section. Faucet currently has two role, it's a tx generator and a Faucet. So I guess:
```yaml
generator:
  interval: 1
```
etc... would be better
2) Can we come up with slightly more descriptive names ?